website/src/components/Banner/index.tsx

```diff
@@ -1,23 +1,18 @@
-import { HiXMark } from "react-icons/hi2";
-import { Banner as FlowbiteBanner, BannerCollapseButton } from "flowbite-react";
-
-export default function Banner({
-  active,
-  overlay,
-  children,
-}: {
-  active: boolean;
-  overlay?: boolean;
-  children: React.ReactNode;
-}) {
-  if (!active) return null;
```

Comment:
If you remove this check then the Banner will be unconditionally rendered on all pages.